front/src/components/boxs/device-in-room/device-features/MultiLevelDeviceFeature.jsx

```diff
@@ -40,7 +40,7 @@ const MultiLevelDeviceType = ({ children, ...props }) => {
             value={props.deviceFeature.last_value}
             onChange={updateValue}
             class="form-control custom-range"
-            step="1"
+            step={props.deviceFeature.step}
```

Comment:
You are adding a step attribute to the deviceFeature object, but I don't see it in the data model. It's not stored?

Those changes affects the whole Gladys, not just the Zwave integration.

Reply:
Reverted